### PR TITLE
Edit/app setup

### PR DESCRIPTION
**Pull request by Pma913**
# PR Template

### Types of changes made?

- [x]  javascript

### What does this PR do?

- This PR cleans out the App component and initializes and empty class component for class and implement Router within the application

### Questions/Comments/Relevant Screenshots?

When reporting a bug, include:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^5.3.0",
```
Comment (hjawad22) [inline package.json:11]: Looks like I did not push up the changes for react-router so we will not have any issues! Thanks for setting this up.

**Comment by hjawad22** [comment]
Thanks for setting up the bare bones! Looks great.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,25 @@
...
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+# Pet Furriendly
+
+## Abstract
```
Comment (tialaaa) [inline README.md:3]: Thanks for starting the ReadMe!

File: package.json (hunk 1 above)
Comment (Pma913) [reply package.json:11]: No problem. Just wanted to clear out the initial clutter so we could all start with a clean slate

File: README.md (hunk 2 above)
Comment (Pma913) [reply README.md:3]: Last minute decision, thought it would be nice

**Pull request merged**